doc/guide/Guide_Installation.ipynb

```diff
@@ -29,35 +29,39 @@
     "\n",
     "### Conda\n",
     "\n",
-    "It's highly recommanded to use [Conda](https://docs.conda.io/projects/conda/en/latest/) for the installation. Without Conda the installation of the dependencies can be cumbersome. If it is not already installed, download the latest version of [Anaconda](https://www.anaconda.com/) or [Miniconda](https://conda.io/miniconda.html) and execute it.\n",
+    "It's highly recommanded to install CLIMADA into a [Conda](https://docs.conda.io/projects/conda/en/latest/) environment. Without Conda, the installation of the dependencies can be cumbersome. If it is not already installed, download the latest version of [Anaconda](https://www.anaconda.com/) or [Miniconda](https://conda.io/miniconda.html) and execute it.\n",
```

Comment:
Here, I changed the wording because it seems like some people think this means "It's highly recommended to use Conda [instead of pip] for the installation". Following that, people seem to wonder about the use of `pip install -e` in the subsection below.